Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
-gem 'cocoapods', '1.7.0.beta.3'
+gem 'cocoapods', '~> 1.6.1'
```

Comment:
#pp don't use optimistic version specifier in conjunction with absolute a.b.c version (that's what the lock file is for).

Reply:
IIRC
`~> 1.6.1` can update to any 1.6.x version, higher than 1.6.1
`~> 1.6` can update to any 1.x.x version, including 1.7.0

So with `~> 1.6.1` the lock file would be pinning 1.6.1 but allow `bundle update` to update the lock file to 1.6.2 if it becomes available, but not 1.7.0

According to [some StackOverflow post](https://stackoverflow.com/a/4292957):

> `~>0.8.5` is semantically equivalent to:
> `gem "cucumber", ">=0.8.5", "<0.9.0"`